src/js/components/Button/BusyAnimation.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled, { css, keyframes } from 'styled-components';
+import { FormCheckmark } from 'grommet-icons/icons/FormCheckmark';
+import { Box } from '../Box';
+
+const bounceDelay = keyframes`
+  0%, 80%, 100% { 
+    transform: scale(0.4);
+  } 40% { 
+    transform: scale(0.8);
+  }
+`;
+
+const bounceDelayRule = css`
+  animation: ${bounceDelay} 1.4s infinite ease-in-out both;
+`;
+
+const Dot = styled(Box)`
+  width: 8px;
+  height: 8px;
+  background-color: ;
+  ${(props) => props.color && `background-color: ${props.color};`}
+  border-radius: 100%;
+  display: inline-block;
+  ${bounceDelayRule}
+  ${(props) => props.delay && `animation-delay: ${props.delay};`}
+`;
+
+export const EllipsisAnimation = ({ color }) => (
+  <Box
+    style={{ position: 'absolute' }}
+    fill
+    alignContent="center"
+    justify="center"
+  >
+    <Box alignSelf="center" direction="row" gap="small">
+      <Dot color={color} delay="-0.32s" />
+      <Dot color={color} delay="-0.16s" />
+      <Dot color={color} />
+    </Box>
+  </Box>
+);
+
+const grow = keyframes`
+  0% {
+    opacity: 0;
+    transform: scale(.3);
+  }
+  20% {
+    opacity: 1;
+    transform: scale(1.15);
+  }
+  30% { transform: scale(.9); }
+  45% { transform: scale(1.05); }
+  55% { transform: scale(1); }
+  100% { transform: scale(1); }  
+`;
+
+export const GrowCheckmark = styled(FormCheckmark)`
```

Comment:
We've been talking about leaning away from use of "Form" icons --> wondering if we should use the normal "Checkmark" here. @ericsoderberghp

Reply:
I tried using the regular 'Checkmark' icon but the sizing looks off. Here is the default size:
<img width="154" alt="Screen Shot 2023-03-03 at 9 38 22 AM" src="https://user-images.githubusercontent.com/54560994/222776521-d66cb8f4-3065-4f5a-a913-430bd36f7cb8.png">

And here is the small size:
<img width="159" alt="Screen Shot 2023-03-03 at 9 45 06 AM" src="https://user-images.githubusercontent.com/54560994/222778000-7d521444-57b9-4333-8f2f-a912fbc14278.png">

This is what is looks like with FormCheckmark:
<img width="155" alt="Screen Shot 2023-03-03 at 9 46 26 AM" src="https://user-images.githubusercontent.com/54560994/222778271-3bc404dd-9950-453a-abd8-fbaa9d3f1cee.png">

Reply:
I think we should ensure it looks reasonable with the grommet theme as well.

For the HPE theme, this will be resolved with the icon sizing work Taylor is doing.

Reply:
Here are the different options in the grommet theme:

Checkmark medium size:
<img width="185" alt="Screen Shot 2023-03-06 at 12 36 23 PM" src="https://user-images.githubusercontent.com/54560994/223212728-c5d9def6-ba15-4b58-a073-bc9a023ef8ac.png">

Checkmark small size:
<img width="196" alt="Screen Shot 2023-03-06 at 12 38 10 PM" src="https://user-images.githubusercontent.com/54560994/223213094-b0a7661e-668e-400e-9dcf-8101756739d3.png">


FormCheckmark:
<img width="199" alt="Screen Shot 2023-03-06 at 12 35 25 PM" src="https://user-images.githubusercontent.com/54560994/223212518-c098b4d5-c299-454a-977b-b1c140d863a3.png">

Reply:
I tested with the grommet-theme-hpe next-stable and grommet-icons stable. Here is the `Checkmark` icon with the default size:

<img width="165" alt="Screen Shot 2023-03-08 at 4 30 19 PM" src="https://user-images.githubusercontent.com/54560994/223876548-24eab5af-cd37-409d-9009-2fed24330f26.png">
<img width="183" alt="Screen Shot 2023-03-08 at 4 30 12 PM" src="https://user-images.githubusercontent.com/54560994/223876573-0c976654-6c64-4bb4-83c2-7bedb097dd68.png">

This feels reasonable to me